### DISCUSSION TITLE
Typography DEFAULT overwrite not working

### DISCUSSION THREAD

**Discussion by maikelvallinga** [tailwindlabs/tailwindcss#11772] [Help] [answered]
Hello,

We are trying to overwrite default css styling for h1, h2, and h3. According to the documentation this could be done with extending the typography module. We can use the prose classes, but we are unable to apply it into the default html elements.

In our tailwind.config.js we added the following:

```
module.exports = {
    theme: {
        extend: {
            typography: {
                DEFAULT: {
                    css: {
                        h1: {
                            color: "#333",
                            fontSize: "200px",
                        },
                        h2: {
                            color: "#333",
                            fontSize: "200px",
                        },
                        h3: {
                            color: "#333",
                            fontSize: "200px",
                        },
                    },
                },
            },
         },
      },
  }
```

But all h1, h2, h3 still don't have the config as we configured. How can we make it work, so the elements will have the settings by default without inserting classes.

**Comment by wongjn** [comment, marked as answer]
Those configuration options would only apply to elements inside an element with the `prose` class name. For elements in general, you could look at adding the rules to your CSS file:

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  h1,
  h2,
  h3 {
    color: #333;
    font-size: 200px;
  }
}
```
Or you could use a Tailwind plugin like:

```js
const plugin = require('tailwindcss/plugin');

/** @type {import('tailwindcss').Config} */
module.exports = {
  // …
  plugins: [
    plugin(({ addBase }) => {
      addBase({
        'h1, h2, h3': {
          color: "#333",
          fontSize: "200px",
        },
      });
    }),
  ]
}
```

**Reply by maikelvallinga** [reply]
Thanks for the fast reply. The first option works by adding it into the CSS.

The second option didn't work here, but since the first does my question is answered 👍 .
